examples/web-instrumentation/src/LogSink.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Accepts log messages from the browser and outputs to the
+ * (server) console.
+ */
+
+/**
+ * @class
+ * @classdesc A collection utility for logs. It is assumed that the logs will
+ * be collected from the output and sent to log storage by a serverless
+ * framework, such as Cloud Run, or that the user is running in a development
+ * environment and wants to see the messages on the console.
+ */
+module.exports = class LogSink {
```

Comment:
I find the name of this class somewhat confusing with the notion of Log Sinks from Stackdriver logging (to get the logs into BigQuery). Could we call it something else like ConsoleLogger or similar?

Reply:
Sure. Changed it to ConsoleLogger.